cli/azd/pkg/infra/provisioning/manager.go

```diff
@@ -86,12 +92,59 @@ func (m *Manager) Deploy(ctx context.Context) (*DeployResult, error) {
 		return nil, fmt.Errorf("updating environment with deployment outputs: %w", err)
 	}
 
+	stAccount, hasBindMountOperation := m.env.LookupEnv(operationsBindMount)
```

Comment:
The operation is right after provision to have it as an extension of IaC and disconnected from any particular service.

The operation is also more general than Aspire or ACA. It can be used regardless of the service.

It is also linked to IaC as a way to express that it is part of the initial state.  We can plan/design/implement more operations in the future for services.